neural_compressor/compression/pruner/patterns/ninm.py

```diff
@@ -145,12 +147,22 @@ def _reshape_orig_to_2dims(self, data):
         Returns:
             Reshaped data.
         """
-        if len(data.shape) == 4:  # TODO: need to verify whether it's ok for transposed conv
+        from neural_compressor.compression.pruner.utils import FLATTEN_DIM2
```

Comment:
from ..utils import FLATTEN_DIM2

Reply:
nice catch, updated in [ed6395b](https://github.com/intel/neural-compressor/pull/1492/commits/ed6395b2fa9ae9aecce68b2a0ceb6e3e1ec2f02d).